Mathlib/GroupTheory/GroupAction/MultipleTransitivity.lean

```diff
@@ -284,6 +292,114 @@ end Higher
 
 end MulAction
 
+namespace Equiv.Perm
+
+variable {α : Type*} [Fintype α]
+
+-- Move?
+/-- The permutation group of `α` acts transitively on `α`. -/
+instance : IsPretransitive (Perm α) α where
+  exists_smul_eq x y := by
+    classical
+    use Equiv.swap x y
+    simp only [Equiv.Perm.smul_def, Equiv.swap_apply_left x y]
+
+/-- The permutation group of `α` acts 2-pretransitively on `α`. -/
+theorem is_two_pretransitive (α : Type*) :
```

Comment:
I guess there already is an `α` in scope.

Reply:
Yes. The point was to make it explicit. I can use (variable () in).